src/main/java/org/broadinstitute/hellbender/utils/genotyper/UnfilledReadsLikelihoods.java

```diff
@@ -0,0 +1,123 @@
+package org.broadinstitute.hellbender.utils.genotyper;
+
+import htsjdk.samtools.util.Locatable;
+import htsjdk.variant.variantcontext.Allele;
+import it.unimi.dsi.fastutil.objects.Object2IntMap;
+import org.broadinstitute.hellbender.utils.IndexRange;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.pileup.PileupElement;
+import org.broadinstitute.hellbender.utils.pileup.ReadPileup;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+//TODO comment this off
+public class UnfilledReadsLikelihoods<A extends Allele> extends ReadLikelihoods<A> {
```

Comment:
I don't like this as a subclass, I feel like it's almost certainly better as a second implementation of an abstract class

Reply:
I will do this in another pass, I have opened a ticket.